src/adapters/rubicon.js

```diff
@@ -286,10 +284,32 @@ var RubiconAdapter = function RubiconAdapter() {
     utils.logMessage('Rubicon Project bidding complete: ' + ((new Date).getTime() - _bidStart));
 
     utils._each(slots, function (slot) {
-      _addBids(slot, slot.getRawResponses());
+      _makeBids(slot, slot.getRawResponses());
     });
   }
 
+
+  var _cb;
+  var _eventAvailable;
+  /**
+   * Used to attach (and switch out) callback for listening to rubicon bid events
+   * Rubicon
+   * @param {Function} cb Callback to register with event handler
+   * @return {Boolean} whether we can handle the event or not
+   */
+  function _handleBidEvent(cb) {
+    _cb = cb;
+    if(_eventAvailable) {
```

Comment:
if( here and if ( two lines below.  Either is find, but consistency would be best.  Find replace all occurences